### ISSUE TITLE
Debugger: Infinite "Starting local session"

### ISSUE THREAD

**Issue by astrelsky** [NationalSecurityAgency/ghidra#2616]
**Describe the bug**
If `DbgEngClientThreadExecutor.init` fails with an exception then the "Starting local session" popup waits forever until cancelled.

**Expected behavior**
If an exception occurs the dialog should close and the user notified of an error.

**Screenshots**

**Attachments**
If applicable, please attach any files that caused problems or log files generated by the software.

**Environment (please complete the following information):**
 - OS: Microsoft Windows [Version 10.0.19042.685]
 - Java Version: 15.0.1
 - Ghidra Version: 9.3_DEV

**Additional context**
The exception being caused is due to #2615

**Comment by nsadeveloper789** [comment]
Agreed. This is an error-handling issue. Are you launching with the "IN-VM" or "local agent over GADP/TCP" variant?

**Comment by astrelsky** [comment]
> Agreed. This is an error-handling issue. Are you launching with the "IN-VM" or "local agent over GADP/TCP" variant?

local agent over GADP/TCP
